release-please-config.json

```diff
@@ -53,6 +53,9 @@
     "packages/instrumentation-ioredis": {},
     "packages/instrumentation-knex": {},
     "packages/instrumentation-koa": {},
+    "packages/instrumentation-langchain": {
+      "skip-github-release": true
```

Comment:
It would be good to ensure this would not be tried to be released as is currently a not fully functional instrumentation, I do not know if this will be sufficient